buffer/src/main/java/io/netty/buffer/PoolChunkList.java

```diff
@@ -122,12 +122,12 @@ private void remove(PoolChunk<T> cur) {
 
     @Override
     public int minUsage() {
-        return minUsage;
+        return minUsage == Integer.MIN_VALUE ? 1 : minUsage;
```

Comment:
`max(1, minUsage)` -> max is intrinsic and less dependency on `Integer.MIN_VALUE` "special value"

http://hg.openjdk.java.net/jdk8/jdk8/hotspot/file/87ee5ee27509/src/share/vm/classfile/vmSymbols.hpp#l633
http://hg.openjdk.java.net/jdk8/jdk8/hotspot/file/87ee5ee27509/src/share/vm/classfile/vmSymbols.hpp#l653